GravityPDF/gravity-pdf-previewer
# Add zoom selector PDF setting

## Description

Adds field zoom controls to the Previewer field in the Gravity Forms Editor. 

Fixes https://github.com/GravityPDF/gravity-pdf-previewer/issues/124

## Testing instructions
1. Add a Previewer field to the form
2. Select the Appearance tab
3. Change the Zoom setting and save
4. Preview the form and the Previewer should default to this zoom level
